strategies/always_blue/always_blue.py

```diff
@@ -22,7 +22,7 @@
 from typing import Dict, Any, List, Union
 
 
-def get_always_blue() -> Dict[str, Union[int, str]]:
+def get_always_blue() -> Dict[str, Union[int, List[str]]]:
     """ALWAYS BLUE."""
     return {"bet_amount": 0, "info": "ALWAYS BLUE!"}
```

Comment:
Logs need to be returned in lists in order to be parsed by the skill correctly:

https://github.com/valory-xyz/trader/pull/201#discussion_r1444925712

Reply:
Ok, I put the info in a list now.